### PR TITLE
fet: l2-bridge-arb iterate over l1

### PR DESCRIPTION
**Pull request by sergeyWh1te**
Screenshot 2024-07-19 at 19 35 44
Screenshot 2024-07-19 at 19 39 26

1. l2-bridge-arb iterates over l1 network
2. It compares bridge balances at the same timestamps on l1 and l2 sides
3. Moved findings from l2-bridge-eth to l2-bridge-arb. In other words, all checks happen here, in l2-bridge-arb 
4. Process all l2 blocks and their logs for detecting events

### CODE REVIEW

File: l2-bridge-arbitrum/src/clients/l2_client.ts [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+            } else {
+              // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+              // @ts-expect-error
+              throw new Error(obj.error.message)
+            }
+          }
+
+          return result
+        },
+        { delay: 750, maxTry: 10 },
```
Comment (TheDZhon) [inline l2-bridge-arbitrum/src/clients/l2_client.ts:192]: Should we introduce constants?

File: l2-bridge-arbitrum/src/db/seeds/l2_withdrawals.sql
Comment (TheDZhon) [inline l2-bridge-arbitrum/src/db/seeds/l2_withdrawals.sql:1]: do we need seeds for l2 withdrawals?

**Review by TheDZhon — APPROVED** [review]
:+1:  left a few small comments

File: l2-bridge-arbitrum/src/db/seeds/l2_withdrawals.sql
Comment (sergeyWh1te) [reply l2-bridge-arbitrum/src/db/seeds/l2_withdrawals.sql:1]: Seeds are needed for tests.

**Pull request merged**